packages/charts/react-charting/src/components/LineChart/LineChart.base.tsx

```diff
@@ -820,40 +820,79 @@ export class LineChartBase extends React.Component<ILineChartProps, ILineChartSt
           pointsForLine.push(
-            <path
-              id={circleId}
-              key={circleId}
-              d={path}
-              data-is-focusable={isLegendSelected}
-              onMouseOver={this._handleHover.bind(
-                this,
-                x1,
-                y1,
-                verticaLineHeight,
-                xAxisCalloutData,
-                circleId,
-                xAxisCalloutAccessibilityData,
-              )}
-              onMouseMove={this._handleHover.bind(
-                this,
-                x1,
-                y1,
-                verticaLineHeight,
-                xAxisCalloutData,
-                circleId,
-                xAxisCalloutAccessibilityData,
-              )}
-              onMouseOut={this._handleMouseOut}
-              onFocus={() => this._handleFocus(lineId, x1, xAxisCalloutData, circleId, xAxisCalloutAccessibilityData)}
-              onBlur={this._handleMouseOut}
-              {...this._getClickHandler(this._points[i].data[j - 1].onDataPointClick)}
-              opacity={isLegendSelected && !currentPointHidden ? 1 : 0.01}
-              fill={this._getPointFill(lineColor, circleId, j, false)}
-              stroke={lineColor}
-              strokeWidth={strokeWidth}
-              role="img"
-              aria-label={this._getAriaLabel(i, j - 1)}
-            />,
+            this.props.lineMode === 'scatter' ? (
+              <circle
+                id={circleId}
+                key={circleId}
+                r={this._points[i].data[j - 1].markerSize ?? 4}
+                cx={this._xAxisScale(x1)}
+                cy={this._yAxisScale(y1)}
+                data-is-focusable={isLegendSelected}
+                onMouseOver={this._handleHover.bind(
+                  this,
+                  x1,
+                  y1,
+                  verticaLineHeight,
+                  xAxisCalloutData,
+                  circleId,
+                  xAxisCalloutAccessibilityData,
+                )}
+                onMouseMove={this._handleHover.bind(
+                  this,
+                  x1,
+                  y1,
+                  verticaLineHeight,
+                  xAxisCalloutData,
+                  circleId,
+                  xAxisCalloutAccessibilityData,
+                )}
+                onMouseOut={this._handleMouseOut}
+                onFocus={() => this._handleFocus(lineId, x1, xAxisCalloutData, circleId, xAxisCalloutAccessibilityData)}
+                onBlur={this._handleMouseOut}
+                {...this._getClickHandler(this._points[i].data[j - 1].onDataPointClick)}
+                opacity={isLegendSelected && !currentPointHidden ? 1 : 0.01}
+                fill={this._getPointFill(lineColor, circleId, j, false)}
+                stroke={lineColor}
+                strokeWidth={strokeWidth}
+                role="img"
+                aria-label={this._getAriaLabel(i, j - 1)}
+              />
+            ) : (
+              <path
```

Comment:
Can't we modify the path (d, fill, and stroke) to display markers? In LineChart, we support legend shapes other than circles, so this might be useful for scatter plots in the future as well. Something to consider as a follow-up.